### DISCUSSION TITLE
Asking an idea of setup a develop environment in Peppol

### DISCUSSION THREAD

**Discussion by tonytram** [phax/phase4#290] [Q&A] [answered]
Dear Philip,

Currently, I'm using Phase4 for Peppol profile with just 2 environments which are TEST and PROD. Is there any good idea to setup a DEV environment for developer testing purposes?

Thank you for your support,

**Comment by phax** [comment, marked as answer]
Hi Tony. Thanks for asking.
Well, the Peppol infrastructure is setup in 2 stages: Production and Test/Pilot.
It means, we have:
* SML and SMK
* AP Production and AP Test PKI
* SMP Production and SMP Test PKI
* Production Directory and Test Directory
* Two different DNS domains we are querying

So from that perspective 2 stages is all we have. If you need a third stage in your application, I assume you need the "Test/Pilot" stage of the Peppol Network.

As this is application-dependent, I leave that up to you.
Does that make sense?

**Comment by tonytram** [comment]
Dear Philip,

Yes, it makes sense, because we stick to Peppol Network environment. 

Thank you for your quick response.
